mixer/cmd/bundles.go

```diff
@@ -0,0 +1,76 @@
+// Copyright © 2017 NAME HERE <EMAIL ADDRESS>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/clearlinux/mixer-tools/builder"
+
+	"github.com/spf13/cobra"
+)
+
+type bundleCmdFlags struct {
+	all   bool
+	force bool
+	git   bool
+}
+
+var bundleFlags bundleCmdFlags
+
+var addBundlesCmd = &cobra.Command{
+	Use:   "add-bundles",
```

Comment:
Should be `Use:    "add-bundles [bundle list]"` or something to reflect the arg.